setup.h.in

```diff
@@ -545,7 +547,7 @@
 #define wxUSE_DRAG_AND_DROP 0
 
 #ifdef __WXMSW__
-#define wxUSE_ACCESSIBILITY 1
+#define wxUSE_ACCESSIBILITY 0
```

Comment:
Thanks for noticing this! It looks like this `1` was introduced manually in 8b5387bc3da33f437a53edcb11847802b628eb70 because this was done by regenerating this auto-generated file. This doesn't really matter because `setup.h.in` is only used with configure which will set this symbol correctly anyhow, but I'll separate this change from the other commits to keep things clean.